test/otel/testdata/ocb_build_script.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+mkdir -p /tmp/otel-ci
```

Comment:
I'd say we clean up `/tmp/otel-ci` after the test runs, even despite being `/tmp`.